specification/servicebus/resource-manager/Microsoft.ServiceBus/stable/2017-04-01/servicebus.json

```diff
@@ -3986,6 +4210,61 @@
         }
       },
       "description": "The result of the List Alias(Disaster Recovery configuration) operation."
+    },
+	"MigrationConfigProperties": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "properties": {
+            "provisioningState": {
+              "readOnly": true,
+              "type": "string",
```

Comment:
provisioningState is a readonly field returned from server and may have additional values in the future. Modeling it as enums will cause client side runtime exceptions if there is a different value received.

Reply:
made it read only string